marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/reader.spec.tsx

```diff
@@ -96,6 +96,15 @@ describe("Entity Modeling: Reader Role", () => {
   it("can navigate to graph view from table view", () => {
     entityTypeTable.viewEntityInGraphView("Customer");
 
+    //To test graph view model png is downloaded successfully
+    graphView.getExportGraphIcon().click();
+    cy.wait(1000);
```

Comment:
Maybe instead of this hard wait you can wait for something in the UI, and place the cy.readfile inside a ".then()" (let me know if you need help with this)

Reply:
Done!